src/sympc/module/nn/functional.py

```diff
@@ -24,6 +26,9 @@ def relu(x: MPCTensor) -> MPCTensor:
     Returns:
         An MPCTensor which represents the ReLu applied on the input tensor
     """
+    relu_forward = GRAD_FUNCS.get("relu", None)
+    if relu_forward and x.session.autograd_active:
+        return forward(x, relu_forward)
```

Comment:
this will the issue for not recording the grad right? but I wonder where is it being stored as node ?

Reply:
It is recorded - the forward function deals with this.